internal/profile/_static/kibana_config_default.yml

```diff
@@ -17,5 +17,6 @@ xpack.fleet.registryUrl: "https://package-registry:8080"
 xpack.fleet.agents.enabled: true
 xpack.fleet.agents.elasticsearch.host: "https://elasticsearch:9200"
 xpack.fleet.agents.fleet_server.hosts: ["https://fleet-server:8220"]
+xpack.fleet.enableExperimental: ["experimentalDataStreamSettings"] # Enable experimental toggles in Fleet UI
```

Comment:
Does it enable the features or only the toggles?

Reply:
Just the toggles are enabled - the features themselves still require user input or package spec opt-in by the integration.